gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -67,97 +66,138 @@
-            .build();
-
-    Layer layer =
-        Layer.newBuilder()
-            .setId("layer id")
-            .setItemLabel("item label")
-            .setListHeading("heading title")
-            .setDefaultStyle(Style.builder().setColor("000").build())
-            .setForm(form)
-            .build();
-
-    Project.Builder builder =
-        Project.newBuilder()
-            .setId("project id")
-            .setTitle("project 1")
-            .setDescription("foo description");
-    builder.putLayer(layer.getId(), layer);
-    return builder.build();
-  }
-
-  private static FeatureMutation createTestFeatureMutation(
-      String userId, String projectId, String layerId) {
-    return FeatureMutation.builder()
-        .setId(1L)
-        .setType(Mutation.Type.CREATE)
-        .setUserId(userId)
-        .setProjectId(projectId)
-        .setFeatureId("feature id")
-        .setLayerId(layerId)
-        .setNewLocation(
-            Optional.ofNullable(Point.newBuilder().setLatitude(110.0).setLongitude(-23.1).build()))
-        .setClientTimestamp(new Date())
-        .build();
+  private static void assertObservationMutation(
+      ObservationMutation expected, ObservationMutation actual) {
+    // TODO: Id is auto-assigned to ObservationMutation.
+    //  If we try to give it while inserting, then it causes problems. Improve this behavior.
+    //  So, copy the id from actual to expected and then compare the objects.
+    expected = expected.toBuilder().setId(actual.getId()).build();
+    Assert.assertEquals(expected, actual);
   }
 
-  private static ObservationMutation createTestObservationMutation(
-      String projectId, String featureId, String layerId, String formId, String userId) {
-    return ObservationMutation.builder()
-        .setType(Mutation.Type.CREATE)
-        .setProjectId(projectId)
-        .setFeatureId(featureId)
-        .setLayerId(layerId)
-        .setObservationId("observation id")
-        .setFormId(formId)
-        .setResponseDeltas(
-            ImmutableList.<ResponseDelta>builder()
-                .add(
-                    ResponseDelta.builder()
-                        .setFieldId("field id")
-                        .setNewResponse(TextResponse.fromString("response for field id"))
-                        .build())
-                .build())
-        .setClientTimestamp(new Date())
-        .setUserId(userId)
-        .build();
+  private static void assertObservation(ObservationMutation mutation, Observation observation) {
+    Assert.assertEquals(mutation.getObservationId(), observation.getId());
+    Assert.assertEquals(mutation.getFeatureId(), observation.getFeature().getId());
+    Assert.assertEquals(TEST_USER, observation.getCreated().getUser());
+    Assert.assertEquals(TEST_FORM, observation.getForm());
+    Assert.assertEquals(TEST_PROJECT, observation.getProject());
+    Assert.assertEquals(TEST_USER, observation.getLastModified().getUser());
+    MatcherAssert.assertThat(
```

Comment:
Same here: https://truth.dev/

Please be sure to use static imports to make assertions more fluent.